android/app/src/main/java/updated/mysterium/vpn/model/filter/NodePrice.kt

```diff
@@ -0,0 +1,16 @@
+package updated.mysterium.vpn.model.filter
+
+import updated.mysterium.vpn.model.manual.connect.PriceLevel
+
+enum class NodePrice {
+    LOW, MEDIUM, HIGH;
```

Comment:
Enum values should be always on separate lines.

Reply:
Fix